api/interaction.go

```diff
@@ -78,6 +79,11 @@ type InteractionResponseData struct {
 	//
 	// During all other events, this should not be provided.
 	Choices *[]AutocompleteChoice `json:"choices"`
+	
+	// CustomID used with the modal
+	CustomID option.NullableString `json:"custom_id,omitempty"`
+	// Title is the heading of the modal window
+	Title    option.NullableString `json:"title,omitempty"`
```

Comment:
This line isn't formatted. Can you do that?

Reply:
Sure will